application/models/post.go

```diff
@@ -811,28 +810,37 @@ func (p *Posts) FindByUser(ctx context.Context, user User) error {
 			SELECT id FROM organizations WHERE id IN (
 				SELECT secondary_id FROM organization_trusts WHERE primary_id IN (SELECT id FROM o)
 			)
-		) AND visibility IN (?, ?)
+		) AND visibility = ?
 	)
-	AND status not in (?, ?) ORDER BY created_at desc`,
-		user.ID, PostVisibilityAll, PostVisibilityAll, PostVisibilityTrusted, PostStatusRemoved, PostStatusCompleted)
-	if err := q.All(p); err != nil {
+	AND status not in (?, ?)`
+
+	args := []interface{}{user.ID, PostVisibilityAll, PostVisibilityTrusted, PostStatusRemoved,
+		PostStatusCompleted}
+
+	if searchText != nil {
+		selectClause = selectClause + " AND (LOWER(title) LIKE ? or LOWER(description) LIKE ?)"
+		likeText := "%" + strings.ToLower(*searchText) + "%"
+		args = append(args, likeText)
+		args = append(args, likeText)
```

Comment:
I think you can just include this second `likeText` in the first append.

Reply:
Good.